app/controllers/application_controller/ci_processing.rb

```diff
@@ -203,7 +203,6 @@ def ownership_update
   # Retire 1 or more items (vms, stacks, services)
   def retirevms
     assert_privileges(params[:pressed])
-    vms = find_checked_items
```

Comment:
How is this change related to `Physical Server`?

If it's not related, please avoid making changes like these since it disturbs the flow of the review process.